src/Commands/AutonomousCommand.h

```diff
@@ -23,10 +11,7 @@
  */
```

Comment:
@MajorDarius, are you referring to the end of comment shown in the diff view or the entire comment shown in the file?

Remember that the diff view does not show the entire file. Only the changes to the file are shown in the diff view. 

However, looking at the file, I'm not sure that the comment actually has any value. But we don't need to change it for this PR.